public/js/shared.js

```diff
@@ -119,6 +119,30 @@ function policyValueSelector(policy) {
   `;
 }
 
+/**
+ * Render the not supported banner html string based on policy object given.
+ * @param {!Object} policy
+ * @return {string}
+ */
+function notSupportedBanner(policy) {
+  if ('Document-Policy'.localeCompare(policy.policyType) === 0) {
+    return `
```

Comment:
Return as html, e.g.:
```suggestion
    return html`
```

Reply:
Done.